utils/content.ts

```diff
@@ -277,15 +306,16 @@ export const getContentInCategory = (
         ...data,
         previewImage: data.previewImage || '/images/image-placeholder.png',
         id: uuid(),
-      };
+      } as IContent;
```

Comment:
Added type assertion because for TS the type returned had `previewImage` and `id` but this was wrong because in lines 306-308 we spread the `data` object and added `previewImage` and `id`

Reply:
Interesting, wonder why we need this... What are the properties in the `data` object?

Reply:
The initial type when hovering over data is
```
const data: {
    [key: string]: any;
}
```

These are the properties that `data` can have, some properties are optional
```
  title: string;
  slug: string;
  basePath: string;
  date: Date;
  draft?: boolean;
  selectedWork?: boolean;
  description?: string;
  previewImage?: string;
  tags?: string[];
  category?: string;
```

Reply:
why isn't `data` this type? maybe it should have this type applied to it instead?